Admin/AdminHelper.php

```diff
@@ -201,4 +229,105 @@ public function camelize($property)
     {
         return BaseFieldDescription::camelize($property);
     }
+
+    /**
```

Comment:
Could you add a description, like, "recursively find the class name of the admin responsible for the element at the end of an association chain" ? It's hard to figure out what this method does (I might be wrong about the description, BTW)